modules/nf-core/hisat2/align/main.nf

```diff
@@ -33,6 +33,7 @@ process HISAT2_ALIGN {
     } else if (meta.strandedness == 'reverse') {
         strandedness = meta.single_end ? '--rna-strandness R' : '--rna-strandness RF'
     }
+    ss = "$splicesites" ? "--known-splicesite-infile $splicesites" : ''
```

Comment:
`ss` is way too short as a variable, but I'll blame the modules side, not here the pipeline side

Reply:
Yep, nowt to do with us guv'nor.

Reply:
It's not like we're limited by the number of characters in a string to name a variable